docs/tools/terraform/v1v2.rst

```diff
@@ -0,0 +1,216 @@
+Upgrading the Aiven Terraform Provider from v1 to v2
+====================================================
+
+Version 2 of the Aiven Terraform Provider was released in `October of
+2020 <https://aiven.io/blog/aiven-terraform-provider-v2-release>`__. You
+may still be running v1 and wanting to upgrade, this guide will walk you
+through the process (there are quite a few ways to achieve this
+technically but we will be focusing on only one).
+
+Notable changes
+---------------
+
+-  Billing Groups have been introduced instead of needing to provide
+   Card ID
+-  Work is being done to deprecate ``aiven_service`` in order to support
+   individual service configuration better, using ``aiven_kafka`` for
+   example
+-  New resources has been added to the Aiven Terraform provider, such as
+   ``aiven_flink``, ``aiven_opensearch``, etc.
+-  More details of all of the changes can be found in the
+   `Changelog <https://github.com/aiven/terraform-provider-aiven/blob/master/CHANGELOG.md>`__.
+
+Original setup
```

Comment:
I'm a little confused. Are we walking people through installing an old version here? I think if they are following an upgrade guide, we can assume they already have an old version so we can skip this step completely.

Reply:
Idea here was just to demonstrate potential original setup using v1 provider (which they most probably have) so they could compare it to v2 to see the difference. 

But we can replace it with something like "We assume you already have set some services up using Aiven Terraform provider v1.3+ and Terraform 0.12+". Wdyt?

Reply:
I think that would be good. Just "this guide is for users already using version[whatever] and ready to upgrade" would work. Otherwise I'm following all the steps thinking this is how to upgrade. We could even signpost the project installation instructions in case people end up here somehow without being on v1.